apps/src/sites/studio/pages/levels/show.js

```diff
@@ -23,6 +24,10 @@ function initPage() {
   const codeReviewEnabled = config.code_review_enabled;
   getStore().dispatch(setCodeReviewEnabled(codeReviewEnabled));
 
+  // If viewing the unit overview components on the level page it is in
+  // the mini view
+  getStore().dispatch(setIsMiniView(true));
```

Comment:
For the future: `UnitOverview` already has an `onOverivewPage` property which expresses the same thing.  I see that it would be a pain to pass this prop down through the component tree to where we need it which is probably why we've put data like this in redux in the past.  If/when we get another pass at this, I'd be curious if the new React Context would be a cleaner way to distribute data through the component tree without having to put this type of display state in redux.

Reply:
I was wondering the same thing about React Context but I don't understand it well enough to make this quick fix with it. I agree that would probably be a better route.

Reply:
eventually using React Context in this kind of situation sounds right to me 👍